chaoslib/aws_chaos/chaosutil_aws.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Stop the AWS ec2 instances
```

Comment:
Can we change the task name, as we can perform stop,start,restart,etc.. from the same util?